components/progressbar/stories/progressbar.stories.js

```diff
@@ -18,6 +36,9 @@ export default {
 			table: {
 				type: { summary: "string" },
 				category: "Component",
+
+				// todo: side labels are not supported according to S1 documentation, but will be in S2. Remove the disable: true once S2 is released
```

Comment:
None of the documentation talk about a side label (for [progress bar](https://spectrum.adobe.com/page/progress-bar/) or [meter](https://spectrum.adobe.com/page/meter/)), but we showcase the side label on the CSS docs for [progress bar](https://opensource.adobe.com/spectrum-css/progressbar.html) (although not for meter).

It looks like the [S2 progress bar designs](https://www.figma.com/design/eoZHKJH9a3LJkHYCGt60Vb/S2-token-specs?m=auto&node-id=9602-4210&t=6pAGm0qRO6Z0Ky1W-1) have a few examples, so once `spectrum-two` is merged, we should be able to re-enable this control.